ui/app/components/b64-toggle.js

```diff
@@ -99,6 +99,7 @@ export default Component.extend({
   },
 
   didReceiveAttrs() {
+    this._super();
```

Comment:
is super required for hooks?

Reply:
Yes it should be called. That looks like it was auto fixed by `eslint`. I usually do `this._super(...arguments)` but for this hook there may not be arguments passed in.